pandas/tests/frame/methods/test_to_records.py

```diff
@@ -379,3 +379,12 @@ def test_to_records_datetimeindex_with_tz(self, tz):
 
         # both converted to UTC, so they are equal
         tm.assert_numpy_array_equal(result, expected)
+
+    def test_to_records_timeseries(self):
+        index = date_range("1/1/2000", periods=10)
+        df = DataFrame(np.random.randn(10, 3), index=index, columns=["a", "b", "c"])
+
+        result = df.to_records()
+        result["index"].dtype == "M8[ns]"
+
+        result = df.to_records(index=False)
```

Comment:
can you assert the expected (or something about it)

Reply:
this is just moving the existing test to the correct location.  i can revert the move as out of scope

Reply:
its fine to move, but if you can an expected value would be good

Reply:
reverted this move